app/helpers/job_invocation_output_helper.rb

```diff
@@ -0,0 +1,32 @@
+module JobInvocationOutputHelper
+  CONSOLE_COLOR = {
+      '31' => 'red',
+      '32' => 'lightgreen',
+      '33' => 'orange',
+      '0'  => 'default'
+    }
+  CONSOLE_COLOR.default = 'default'
+
+  def colorize_line(line)
+    line = line.gsub(/\e\[.*?m/) do |seq|
+      color = seq[/(\d+)m/,1]
+      "{{{format color:#{color}}}}"
+    end
+
+    current_color = 'default'
```

Comment:
@adamruzicka I think this is the part that's causing the problem you're seeing where our own output is colorized. The real issue though, is that at this point we have no way of discriminating where the `line` is coming from. From `foreman_remote_execution` POV, it's all `continuous_output`. 

I made `current_color` NOT take the previous value (so after a green line another would follow if there's no color specified) - now all lines are `default` unless a color is specified.

https://github.com/theforeman/foreman_remote_execution/blob/master/app/lib/actions/remote_execution/run_host_job.rb#L106

https://github.com/theforeman/foreman-tasks/blob/master/lib/foreman_tasks_core/continuous_output.rb